jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ResourceServlet.java

```diff
@@ -206,6 +214,23 @@ public void init() throws ServletException
         if (baseResource != null && !(baseResource.isDirectory() && baseResource.isReadable()))
             LOG.warn("baseResource {} is not a readable directory", baseResource);
 
+        if (getInitBoolean("installAliasChecker", true))
+        {
+            // Add a new aliasCheck to the ContextHandler if one does not exist for this baseResource.
+            boolean addAliasCheck = true;
+            for (AliasCheck aliasCheck : contextHandler.getAliasChecks())
+            {
+                if (aliasCheck instanceof AllowedResourceAliasChecker allowedResourceAliasChecker &&
+                    Objects.equals(baseResource, allowedResourceAliasChecker.getBaseResource()))
```

Comment:
Perhaps if the user has added any alias checkers at all, then we should not add a default?

Reply:
This doesn't work because a symlink aliasCheck is normally added by default. And then it will also not work with multiple ResourceServlets for different directories.